azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/ListVirtualMachineExtensionImages.java

```diff
@@ -55,7 +55,7 @@ public static boolean runSample(Azure azure) {
             System.out.println("Publisher - " + publisher.name());
 
             if (publisher.name().equalsIgnoreCase("Microsoft.OSTCExtensions")
-                    | publisher.name().equalsIgnoreCase("Microsoft.Azure.Extensions")) {
```

Comment:
Looks like incorrect logical OR condition.
I guess it should be 'II' and not 'I'

Reply:
[P2]: Nice catch, I marked it as P2 as it is a fix needed in sample. Thanks for finding this.